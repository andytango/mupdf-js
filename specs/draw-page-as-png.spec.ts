import {
  forExamplePages,
  loadExampleFile,
  TEST_DPI,
} from "./helpers/example-file";

describe("drawPageAsPNG", () => {
  forExamplePages("should return a b64 encoded png file", async (page) => {
    const { mupdf, doc } = await loadExampleFile();
    expect(mupdf.drawPageAsPNG(doc, page, TEST_DPI)).toMatchSnapshot();
  });
});
