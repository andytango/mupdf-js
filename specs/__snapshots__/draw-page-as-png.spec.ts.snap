// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`drawPageAsPNG Page [1 / 4] - should return a b64 encoded png file 1`] = `"12a44168274dc87f91e37aa61d66ac41"`;

exports[`drawPageAsPNG Page [2 / 4] - should return a b64 encoded png file 1`] = `"768243e999e4ca3bff7ab935cef3f286"`;

exports[`drawPageAsPNG Page [3 / 4] - should return a b64 encoded png file 1`] = `"cd56037f422fe6a80ecc484d2086be6b"`;

exports[`drawPageAsPNG Page [4 / 4] - should return a b64 encoded png file 1`] = `"925f0c52022f319b246e83948156550d"`;
